src/hooks/useAudioContext.js

```diff
@@ -1,13 +1,14 @@
 import { useEffect, useRef } from "react";
 
+// Shared between all components.
+let audioCtx;
+
 export function useAudioContext() {
-  const audioCtxRef = useRef();
-  if (!audioCtxRef.current) {
+  if (!audioCtx) {
     // Create once.
     console.log("Creating AudioContext");
-    audioCtxRef.current = new AudioContext();
+    audioCtx = new AudioContext();
```

Comment:
Note: in this case you probably don't want to close it in useEffect cause other components might be using. Could count number of references and close when you reach 0.